data/transactions/logic/assembler_test.go

```diff
@@ -455,6 +450,19 @@ func TestAssemble(t *testing.T) {
 	}
 }
 
+var experiments = []uint64{fidoVersion, pairingVersion}
```

Comment:
@jannotti Is the intention to remove `fidoVersion` in a follow on PR closer to the release or is `fidoVersion` incidentally left here?

Reply:
Yeah, I didn't worry about the actual promotion out of experimental, yet.  I was thinking more about making it easy to move pairing up to 8, rather than making fido permanently 7.

Reply:
@jannotti Got it - I opened an issue to have a visual reference for later:  https://github.com/algorand/go-algorand/issues/4127.  Resolving.